src/sbvr-api/permissions.ts

```diff
@@ -1336,31 +1340,34 @@ const checkApiKey = Bluebird.method(
 
 export const customAuthorizationMiddleware = (expectedScheme = 'Bearer') => {
 	expectedScheme = expectedScheme.toLowerCase();
-	return (
-		req: Express.Request,
-		_res?: Express.Response,
-		next?: Express.NextFunction,
-	): Bluebird<void> =>
-		Bluebird.try(() => {
+	return Bluebird.method(
+		async (
+			req: Express.Request,
+			_res?: Express.Response,
+			next?: Express.NextFunction,
+		): Promise<void> => {
 			const auth = req.header('Authorization');
 			if (!auth) {
+				next?.();
 				return;
 			}
 
 			const parts = auth.split(' ');
 			if (parts.length !== 2) {
+				next?.();
 				return;
 			}
 
 			const [scheme, apiKey] = parts;
 			if (scheme.toLowerCase() !== expectedScheme) {
+				next?.();
```

Comment:
All these `next?.()` calls feel fragile -- I think this one specific case is not worth changing away from `try/then`.